lib/src/component_declaration/component_base.dart

```diff
@@ -389,13 +239,6 @@ abstract class UiComponent<TProps extends UiProps> extends react.Component with
     }
   }
 
-  @override
-  @mustCallSuper
-  void componentWillUnmount() {
```

Comment:
This is now part of the DisposableManagerProxy mixin (yay super-calls in mixins!!)